frontend/src/components/common/BulkPermissionResults.vue

```diff
@@ -1,21 +1,101 @@
 <script setup lang="ts">
-import { Dialog } from '@/lib/primevue';
+import { ref } from 'vue';
+import { Column, DataTable, Dialog } from '@/lib/primevue';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+
 // Props
 type Props = {
   results: Array<Object>;
   resource: any;
   resourceType: string;
 };
-const props = withDefaults(defineProps<Props>(), {});
+
+const props = withDefaults(defineProps<Props>(), {
+  results: undefined
+});
 
 const modelValue = defineModel<boolean>({ default: false });
+
+// Exports DataTable results
+const batchResults = ref();
+
+const exportCSV = (event: any) => {
```

Comment:
guess we can remove the argument

Reply:
I had to throw in the redundant param here - the tests don't seem to like how the`@click=exportCSV($event)` in the `<Button>` has one argument but the declaration has 0 params.

Reply:
Fixed in #216.